x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -71,13 +72,40 @@ const i18nTexts = {
         'Go to your logs directory to view the deprecation logs or enable log collecting to see them in the UI.',
     }
   ),
+  deniedPrivilegeTitle: i18n.translate(
+    'xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeTitle',
+    {
+      defaultMessage: `You're missing index privileges`,
```

Comment:
Because a user will see this immediately after enabling the logs, I think we should be extra precise about this warning, so they know that the logs are still on but they're just not able to view them. Can we change the title and body of the callout to this:

> **You require index privileges to analyze the deprecation logs**
> The deprecation logs will continue to be indexed, but you won't be able to analyze them until you have the read index {privilegesCount, plural, one {privilege} other {privileges}} for: {missingPrivileges}

Reply:
Good point @cjcenizal. This copy suggestion LGTM.

Reply:
Good one! I patched it up with https://github.com/elastic/kibana/pull/112420/commits/efe2e207e01cb74f7905f96eaccb3d23747c8671